### PR TITLE
update release notes for Telegraf 1.13.3

### PR DESCRIPTION
**Pull request by kelseiv**
Update Telegraf 1.13.3 release notes

### CODE REVIEW

File: content/telegraf/v1.13/about_the_project/release-notes-changelog.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ menu:
     parent: About the project
 ---
 
+## v1.13.3 [2020-02-04]
+
+### Bug fixes
+
+- Update Kibana (`kibana`) input plugin to support Kibana 6.4 and later.
+- Prevent duplicate TrackingIDs from being returned in queue consumer plugins.
```
Comment (kelseiv) [inline content/telegraf/v1.13/about_the_project/release-notes-changelog.md]: is queue consumer a specific plugin(s)?

File: content/telegraf/v1.13/about_the_project/release-notes-changelog.md [hunk 2]
```diff
@@ -8,6 +8,15 @@ menu:
...
+- Increase support for up to 4096 statistics in the Ethtool (`ethtool`) input plugin.
+- Remove expired metrics even when no new data is sent to the Prometheus Client (`prometheus_client`) output plugin.
```
Comment (kelseiv) [inline content/telegraf/v1.13/about_the_project/release-notes-changelog.md]: where are expired metrics removed from? is there a difference in behavior is new is sent?

File: content/telegraf/v1.13/about_the_project/release-notes-changelog.md (hunk 1 above)
Comment (danielnelson) [reply content/telegraf/v1.13/about_the_project/release-notes-changelog.md]: No, this fix affects the plugins named `*_consumer`: `mqtt_consumer`, `nats_consumer`, etc plus a few others.  This feature isn't very well communicated to users of the affected plugins either, for example it is not indicated in the plugin if it takes advantage of the metric tracking feature.

File: content/telegraf/v1.13/about_the_project/release-notes-changelog.md (hunk 2 above)
Comment (danielnelson) [reply content/telegraf/v1.13/about_the_project/release-notes-changelog.md]: The `prometheus_client` output works by holding on to the metrics sent to it, and displaying them to the client when they access a URL.  So when the metrics are expired they are deleted and will no longer appear when the page is viewed.

Before this fix, old metrics would only be removed when new metrics were added.  If no new metrics are added, they would remain indefinitely.

File: content/telegraf/v1.13/about_the_project/release-notes-changelog.md (hunk 1 above)
Comment (kelseiv) [reply content/telegraf/v1.13/about_the_project/release-notes-changelog.md]: Thank you, @danielnelson. Moving forward, does it make sense to let users know (which?) affected plugins take advantage of the “metric tracking feature”….and document what this feature is/which metrics are tracked?

Comment (danielnelson) [reply content/telegraf/v1.13/about_the_project/release-notes-changelog.md]: Yes for sure, I opened an issue on the Telegraf repo for this: https://github.com/influxdata/telegraf/issues/6986

**Pull request merged**